packages/obonode/obojobo-chunks-multiple-choice-assessment/mc-assessment-explanation.js

```diff
@@ -5,6 +5,8 @@ import React from 'react'
 const { Button } = Common.components
 const { focus } = Common.page
 
+const DEFAULT_READ_EXPLANATION_LABEL = 'Read Explanation'
```

Comment:
Just to highlight for @zachberry - this is changing the default label from `Read an explanation of the answer` to `Read Explanation`.